font-patcher

```diff
@@ -770,10 +793,12 @@ class font_patcher:
                 # Currently stretching vertically for both monospace and double-width
                 scale_ratio_y = self.font_dim['height'] / sym_dim['height']
 
+            overlap = sym_attr['params']['overlap'] if 'overlap' in sym_attr['params'] else 0
```

Comment:
I don't personally like this conditional style but is it common and does PEP8 or other have particular rules regarding?

Reply:
I honestly do not know. And I do not really care, if you like traditional `if/else` more I can change it whereever it is now.

Reply:
Changed.

This comes from my usual style that a work as much with `const` stuff as possible, and then you need to assign the thing you want and can not use an `if` structure like this.
Python is no language I use often.

Reply:
Hmm, and it can not be avoided for more complex Python constructs (i.e. list comprehension)
```python
                sym_dim = get_multiglyph_boundingBox([ symbolFont[g] if g in symbolFont else None for g in group ])
```
That is very hard to write differently and still in a Python way... So I kept it there. Looks super clumsy if done manually.

Reply:
no worries. thanks!